poem-openapi/src/lib.rs

```diff
@@ -113,6 +113,7 @@
 #![deny(private_in_public, unreachable_pub)]
 #![cfg_attr(docsrs, feature(doc_cfg))]
 #![warn(missing_docs)]
+#![recursion_limit = "256"]
```

Comment:
In general, it could be under `cfg_attr`, like: 
`#![cfg_attr(tarpaulin, recursion_limit = "256")]`

However, I see no issues to keep it as general limit, because it doesn't affect result binary, but only compile time.